windows/test-app.js

```diff
@@ -53,24 +72,18 @@ const binaryExtensions = [".png", ".pfx"];
  * @param destPath Destination path.
  * @param replacements: e.g. {'TextToBeReplaced': 'Replacement'}
  */
-function copyAndReplace(
-  srcPath,
-  destPath,
-  relativeDestPath,
-  replacements = {}
-) {
-  const fullDestPath = path.join(destPath, relativeDestPath);
+function copyAndReplace(srcPath, destPath, replacements = {}) {
   if (fs.lstatSync(srcPath).isDirectory()) {
-    if (!fs.existsSync(fullDestPath)) {
-      fs.mkdirSync(fullDestPath);
+    if (!fs.existsSync(destPath)) {
+      fs.mkdirSync(destPath);
     }
     return;
   }
```

Comment:
Do we still need this? I don't see that we call `copyAndReplace()` with a directory.